checker/raw_result.go

```diff
@@ -31,9 +31,48 @@ type RawResults struct {
 	WebhookResults              WebhooksData
 	MaintainedResults           MaintainedData
 	SignedReleasesResults       SignedReleasesData
+	FuzzingResults              FuzzingData
 	LicenseResults              LicenseData
 }
 
+// FuzzerName represents a fuzzing service.
+type FuzzerName int
+
+const (
+	// FuzzerNameCIFuzz is CIFuzz.
+	FuzzerNameCIFuzz FuzzerName = iota
+	// FuzzerNameOSSFuzz is OSSFuzz.
+	FuzzerNameOSSFuzz
+	// FuzzerNameGoBuiltin is the built-in Go fuzzer.
+	FuzzerNameGoBuiltin
+)
+
+// FuzzingData represents different fuzzing done.
+type FuzzingData struct {
+	Fuzzers []Fuzzer
+}
+
+// Fuzzer represent the use of a fuzzer.
+type Fuzzer struct {
+	Job      *WorkflowJob
+	File     *File
```

Comment:
How does this capture the case where `oss-fuzz` is being used to fuzz? Should we add a `Repository` (or similar) field to signify where we found the evidence for fuzzing?

Reply:
How about leaving out these fields then? We can use another PR to add and populate them. It's better than having empty fields with no info.

Reply:
I'll remove them and add a TODO. Thanks

Reply:
I've removed them.